onnxruntime/core/providers/tensorrt/tensorrt_execution_provider.h

```diff
@@ -288,8 +291,9 @@ class TensorrtExecutionProvider : public IExecutionProvider {
   int (*engine_encryption_)(const char*, char*, size_t) = nullptr;
   bool timing_cache_enable_ = false;
   bool force_timing_cache_match_ = false;
-  bool detailed_build_log_ = false;
+  bool detailed_build_log_ = true;
```

Comment:
why changing it to true?

Reply:
That was for testing purpose and I have changed it back. Thanks for pointing that out